docs/kafka/message-browsing-kafka/README.adoc

```diff
@@ -0,0 +1,158 @@
+:access-mgmt-url-kafka: kafka/access-mgmt-kafka/README.adoc
+:metrics-monitoring-url-kafka: kafka/metrics-monitoring-kafka/README.adoc
+:service-binding-url-kafka: kafka/service-binding-kafka/README.adoc
+
+//OpenShift Service Registry
+:product-long-registry: OpenShift Service Registry
+:product-registry: Service Registry
+:registry: Service Registry
+:product-version-registry: 1
+:service-url-registry: https://console.redhat.com/application-services/service-registry/
+:getting-started-url-registry: registry/getting-started-registry/README.adoc
+:quarkus-url-registry: registry/quarkus-registry/README.adoc
+:rhoas-cli-getting-started-url-registry: registry/rhoas-cli-getting-started-registry/README.adoc
+:access-mgmt-url-registry: registry/access-mgmt-registry/README.adoc
+:content-rules-registry: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1/guide/9b0fdf14-f0d6-4d7f-8637-3ac9e2069817[Supported Service Registry content and rules]
+:service-binding-url-registry: registry/service-binding-registry/README.adoc
+
+//OpenShift Connectors
+:product-long-connectors: OpenShift Connectors
+:service-url-connectors: https://console.redhat.com/application-services/connectors
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-browsing-messages"]
+= Browsing messages in the {product-long-kafka} web console
+ifdef::context[:parent-context: {context}]
+:context: browsing-messages
+
+// Purpose statement for the assembly
+[role="_abstract"]
+
+As a developer or administrator, you can use the {product-long-kafka} web console to view and inspect messages for a Kafka topic. You might use this functionality, for example, to verify that a client is producing messages to the expected topic partition, that your topic is storing messages correctly, or that messages have the expected content.
+
+When you select a topic in the console, you can use the *Messages* tab to view a list of messages for that topic. You can filter the list of messages in the following ways:
+
+* Specify a partition and see messages sent to the partition.
+* Specify a partition and offset. See messages sent to the partition from that offset.
+* Specify a partition and a timestamp (that is, date and time) value. See messages sent to the partition from that date and time.
+* Specify a partition and a Unix epoch timestamp value. See messages sent to the partition from that epoch timestamp value.
+
+//Additional line break to resolve mod docs generation error.
+
+[id="proc-browsing-messages-for-a-topic_{context}"]
+== Browsing messages for a topic
+
+The following procedure shows how to filter and inspect a list of messages for a topic in the {product-kafka} web console.
+
+.Prerequisites
+
+* You have a Kafka instance with a topic that contains some messages. To learn how to create your _first_ Kafka instance and topic and then send messages to the topic that will appear on the *Messages* page, see the following guides:
```

Comment:
Strictly preference, but I tend to avoid using formatting (ital or bold) to emphasize words in formal documentation. I definitely see the purpose of "your *first* Kafka ...", but I find that almost always it's not necessary and sometimes can even come off as unprofessional. I'm not saying that's the case here, and actually do see the value in it. Just mentioning that I don't personally see the necessity for it. Please disregard if you disagree, not objective.

Reply:
Hmmm, unlike bolding terms, I see this as a fairly gentle way of placing emphasis, and I see value in it, when it's used judiciously. I might actually keep this one. It was actually one of the things I noted for us to discuss as a team in our style discussions! :)

Reply:
Sure, sounds good.